src/python/botan.py

```diff
@@ -15,7 +16,7 @@
 """
 Module initialization
 """
-botan = CDLL('libbotan-1.11.so')
+botan = CDLL('libbotan-1.11.dylib')
 
```

Comment:
Here’s an update PR: https://github.com/randombit/botan/pull/247
It fixes the issue introduced by the previous one (#246).

Here’s a copy of the PR comment:


Fixed introduced Mac-specific change, returning the patch to platform-independent.

Original patch was to improve clarity and readability of the test/demo output provided by botan.py.

I don’t think more details are needed for such a small (and fairly obvious) patch?


On Aug 12, 2015, at 2:38 , Simon Warta <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/python/botan.py<https://github.com/randombit/botan/pull/246#discussion_r36830343>:

> @@ -15,7 +16,7 @@
>  """
>  Module initialization
>  """
> -botan = CDLL('libbotan-1.11.so')
> +botan = CDLL('libbotan-1.11.dylib')
>


I think this should be somehow platform independent. Does that work in Windows as well?

—
Reply to this email directly or view it on GitHub<https://github.com/randombit/botan/pull/246/files#r36830343>.

--
Uri Blumenthal
uri@mit.edu<mailto:uri@mit.edu>